src/shared/ContentsTable/TableEntries/PullEntries/PullDirEntry.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  PathContentsFilters,
+  usePrefetchPullDirEntry,
+} from 'services/pathContents/pull/dir'
+
+import DirEntry from '../BaseEntries/DirEntry'
+
+interface PullDirEntryProps {
+  pullId: string
+  name: string
+  urlPath?: string
+  filters?: PathContentsFilters
+}
+
+function PullDirEntry({ pullId, urlPath, name, filters }: PullDirEntryProps) {
+  const flags =
+    filters?.flags && filters?.flags?.length > 0 ? filters?.flags : []
```

Comment:
I think we don't need the first filters?.flags since we're doing filters?.flags?.length